src/Components/Server/src/Circuits/CircuitPrerenderer.cs

```diff
@@ -26,8 +28,12 @@ public async Task<IEnumerable<string>> PrerenderComponentAsync(ComponentPrerende
                 GetFullUri(context.Request),
                 GetFullBaseUri(context.Request));
 
+            // We don't need to unsubscribe because the circuit host object is scoped to this call.
+            circuitHost.UnhandledException += CircuitHost_UnhandledException;
```

Comment:
On second look, I don't think this is correct. The circuit will live longer than the request and when the signalr connection gets stablished you don't want to have this handler throwing.

If you take a look at how i'm doing this on my PR
https://github.com/aspnet/AspNetCore/pull/7954/files#diff-d33c04a2c673816a9a5c6e959a8e5270R91

I'm specifically basing it on whether I'm prerrendering or not.

You can keep this if you want to, but I'm gonna have to change it on my PR so it just adds work to me :(

I'll leave it up to yo.

Reply:
I don't understand this, or how to do extra work to support something that isn't implemented yet, so I'm going to go ahead and merge this 😆